README.md

```diff
@@ -45,11 +44,11 @@ Please read our [troubleshooting guide](/doc/troubleshooting/README.md).
 
 ## Administrator Considerations
 
-Installer behavior can be different depending on whether you are running **winget** with administrator privileges.
+Installer behavior can be different depending on whether you are running **Winget** with administrator privileges.
```

Comment:
This one should be "Pascal" case. The product name when displayed as a single word should be "WinGet".